src/order_cmd.cpp

```diff
@@ -820,6 +820,9 @@ CommandCost CmdInsertOrder(DoCommandFlag flags, VehicleID veh, VehicleOrderID se
 			if (new_order.GetDepotOrderType() & ~(ODTFB_PART_OF_ORDERS | ((new_order.GetDepotOrderType() & ODTFB_PART_OF_ORDERS) != 0 ? ODTFB_SERVICE : 0))) return CMD_ERROR;
 			if (new_order.GetDepotActionType() & ~(ODATFB_HALT | ODATFB_NEAREST_DEPOT | ODATFB_UNBUNCH)) return CMD_ERROR;
 			if ((new_order.GetDepotOrderType() & ODTFB_SERVICE) && (new_order.GetDepotActionType() & ODATFB_HALT)) return CMD_ERROR;
+
+			/* Check if we're allowed to have a new unbunching order. */
+			if ((new_order.GetDepotActionType() & ODATFB_UNBUNCH) && (v->HasFullLoadOrder() || v->HasUnbunchingOrder() || v->HasConditionalOrder())) return CMD_ERROR;
```

Comment:
This would return a generic `ERR_UNKNOWN` for AIs once unbunch is implemented for them. Could these return each their own error string, similar to that of `CmdModifyOrder` below?

Reply:
Sure, I can do that.